src/www/components/popup/PopupComp.scss

```diff
@@ -0,0 +1,234 @@
+@import "../../vars.scss";
+
+.PopupTrigger {
```

Comment:
This section should be moved to `TeamsPage.scss` since it refers to the button that's pressed to open the popup.
(also don't forget the corresponding sections in the media queries)